CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # go-ipfs changelog
 
+## v0.9.1 2021-07-20
+
+This is a small bug fix release resolving the following issues:
+1. A regression where the empty CID bafkqaaa could not resolve on gateways [#8230](https://github.com/ipfs/go-ipfs/issues/8230)
+2. A panic on OpenBSD [#8211](https://github.com/ipfs/go-ipfs/issues/8211)
+3. High CPU usage with QUIC [#8256](https://github.com/ipfs/go-ipfs/issues/8256)
+4. High memory usage with TCP [#8219](https://github.com/ipfs/go-ipfs/issues/8219)
+5. Some pubsub issues ([libp2p/go-libp2p-pubsub#427](https://github.com/libp2p/go-libp2p-pubsub/pull/427), [libp2p/go-libp2p-pubsub#430](https://github.com/libp2p/go-libp2p-pubsub/pull/430))
+6. Updated WebUI to [v2.12.4](https://github.com/ipfs/ipfs-webui/releases/tag/v2.12.4)
+7. Fixed the snap deployment [#8212](https://github.com/ipfs/go-ipfs/pull/8212)
```

Comment:
Added a minimal changelog. Decided to skip the `mkreleaselog` steps here. Seem reasonable @Stebalien?

Reply:
Seems reasonable. (although I'd still consider including contributors).

Also note: I assume mkreleaslog is going to be pretty small.

Reply:
I updated it, it's pretty long actually because of QUIC related things. WDYT?